houston/ardu/command_factory.py

```diff
@@ -0,0 +1,95 @@
+import os
+import logging
+import yaml
```

Comment:
Super nitpick (sorry!): It's considered a best practice to separate your import into three parts: standard libraries (e.g., `os`, `logging`, `typing), external libraries (e.g., `yaml`, `attr`, `bugzoo`), internal modules (e.g., `from .command import Command`). In this case, `yaml` should go into its own section.